dotnet-desktop-guide/net/winforms/printing/how-to-print-in-windows-forms-using-print-preview.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Print Using Print Preview
+description: Learn how to add print preview services to your application by using the Windows Forms PrintPreviewDialog control.
+ms.date: "05/02/2022"
+dev_langs: 
+  - "csharp"
+  - "vb"
+helpviewer_keywords: 
+  - "printing [Windows Forms], using print preview"
+  - "printing [Windows Forms], with print preview"
+  - "print preview"
+ms.custom: devdivchpfy22
+---
+
+# How to print in Windows forms using print preview
```

Comment:
This is missing the (Windows Forms .NET) ending and probably doesn't need to have "Windows forms" in it since it will be at the end between ( )